### PR TITLE
Reorganize structure of IAM docs

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

**Comment by bre-fitzgerald** [comment]
I'm 99.9% sure I caught all the little redirects and link changes, but would appreciate a quick click-through for that last 0.1% 🙏

File: _docs/_user_guide/message_building_by_channel/in-app_messages.md [hunk 1]
```diff
@@ -15,18 +15,15 @@ guide_featured_list:
...
-- name: "Create an In-App Message"
-  link: /docs/user_guide/message_building_by_channel/in-app_messages/create/
-  fa_icon: fas fa-mobile-alt
 - name: "Drag & Drop Editor"
-  link: /docs/user_guide/message_building_by_channel/in-app_messages/drag_and_drop
-  fa_icon: fas fa-paint-brush
+  link: /docs/user_guide/message_building_by_channel/in-app_messages/drag_and_drop/
+  fa_icon: fas fa-mobile-alt
+- name: "Traditional Editor"
+  link: /docs/user_guide/message_building_by_channel/in-app_messages/traditioanl/
```
Comment (KellieHawks) [inline _docs/_user_guide/message_building_by_channel/in-app_messages.md:22]: Broken link here

File: _docs/_user_guide/message_building_by_channel/in-app_messages/traditional.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+guide_featured_title: "Section Articles"
+guide_featured_list:
+- name: Creating an In-App Message
+  link: /docs/user_guide/message_building_by_channel/in-app_messages/traditional/create/
+  fa_icon: fas fa-mobile-alt
+- name: Customize
+  link: /docs/user_guide/message_building_by_channel/in-app_messages/traditional/customize/
+  fa_icon: fas fa-paintbrush
+- name: Templates
```
Comment (KellieHawks) [inline _docs/_user_guide/message_building_by_channel/in-app_messages/traditional.md:19]: Templates opens to a blank page. This should probably link directly to simple survey

File: assets/js/broken_redirect_list.js [hunk 3]
```diff
@@ -701,4 +701,15 @@ validurls['/docs/user_guide/onboarding_with_braze/email_setup/ssl/ssl_clicktrack
 validurls['/docs/user_guide/engagement_tools/campaigns/ideas_and_strategies/campaigns_in_multiple_languages/'] = '/docs/user_guide/engagement_tools/campaigns/ideas_and_strategies/localizing_a_campaign/';
 validurls['/docs/user_guide/engagement_tools/canvas/create_a_canvas/branching/'] = '/docs/user_guide/engagement_tools/canvas/create_a_canvas/create_a_canvas/';
 validurls['/docs/user_guide/engagement_tools/testing/multivariant_testing/#recommended-follow-ups'] = '/docs/user_guide/engagement_tools/testing/multivariant_testing/multivariate_analytics/#recommended-follow-ups';
-validurls['/docs/developer_guide/platform_integration_guides/swift/live_activities'] = '/docs/developer_guide/platform_integration_guides/swift/live_activities/live_activities';
\ No newline at end of file
+validurls['/docs/developer_guide/platform_integration_guides/swift/live_activities'] = '/docs/developer_guide/platform_integration_guides/swift/live_activities/live_activities';
```
Comment (KellieHawks) [inline assets/js/broken_redirect_list.js:704]:
```suggestion
validurls['/docs/developer_guide/platform_integration_guides/swift/live_activities'] = '/docs/developer_guide/platform_integration_guides/swift/live_activities/live_activities/';
```

File: _docs/_user_guide/message_building_by_channel/in-app_messages/traditional.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  link: /docs/user_guide/message_building_by_channel/in-app_messages/traditional/templates/
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/message_building_by_channel/in-app_messages/traditional.md:20]:
```suggestion
  link: /docs/user_guide/message_building_by_channel/in-app_messages/traditional/templates/simple_survey/
```

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**